src/components/WalletDropdown/MiniPortfolio/index.tsx

```diff
@@ -20,6 +21,12 @@ const Wrapper = styled(Column)`
   flex-direction: column;
   height: 100%;
   gap: 12px;
+
+  ${PortfolioRowWrapper} {
+    &:hover {
+      background: ${({ theme }) => theme.hoverDefault};
```

Comment:
confirm the `background` value change will trigger the `background-color` transition on `PortfolioRowWrapper`

Reply:
yep it still works